emperor/util.py

```diff
@@ -532,3 +532,45 @@ def sanitize_mapping_file(data, headers):
 
     return out_lines[1::], out_lines[0]
 
+def guess_coordinates_files(dir_path):
+    """Given a directory return the file paths that can contain coordinates
+
+    Parameters
+    ----------
+    dir_path : str
+        path to the directory where coordinate files are contained
+
+    Returns
+    -------
+    list of str
+        list of filepaths pointing to the coordinates files
+
+    Notes
+    -----
+    If a path inside dir_path meets any of the following criteria, it will be
+    ignored:
+    - Is a hidden file
+    - Is named `Icon?`.
+    - Is folder
+    - Is part of the procrustes results from QIIME, see
+      transform_coordinate_matrices.py
+    """
+    coord_fps = []
+
+    for filepath in listdir(dir_path):
```

Comment:
That probably makes sense, but since that would affect the `-i` option of `make_emperor.py` and this PR is only trying to fix one bug, would you mind opening a separate issue? I'm still not super convinced about the glob though, remember that we have to maintain compatibility with QIIME 1.9.0 and I'm worried that changing `-i` would break something.